### PR TITLE
Feature/add movie to existed list

### PR DESCRIPTION
**Pull request by pink-bunny**

### CODE REVIEW

File: src/pages/Movie/TopInfo/AddToListPopover/component.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      content={(
+        <>
+          <div>
+            <Button type="link">Create new list ...</Button>
+          </div>
+          {list && list.map((item) => (
+            <div key={item.id}>
+              <Button
+                type="link"
+                onClick={() => addToList(item.id)}
```
Comment (plzen) [inline src/pages/Movie/TopInfo/AddToListPopover/component.js]: Don't use inline functions inside components, you can refactor this with creation separate container for button and handle onClick in the button's container

File: src/pages/Movie/TopInfo/AddToListPopover/container.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      popoverVisible: false
+    };
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    const prevVisibleStatus = prevState.popoverVisible;
+    const currentVisibleStatus = !prevVisibleStatus;
```
Comment (plzen) [inline src/pages/Movie/TopInfo/AddToListPopover/container.js]: Refactor this as we discussed today

File: src/pages/Movie/TopInfo/AddToListPopover/component.js (hunk 1 above)
Comment (pink-bunny) [reply src/pages/Movie/TopInfo/AddToListPopover/component.js]: Then we will have more than 3 levels and it is against the [component guide](https://docs.google.com/document/d/1Cvs9QLqpg9TWHygMETVU8ukdd2JXVogwTPeL0cA6ypA/edit).

Comment (pink-bunny) [reply src/pages/Movie/TopInfo/AddToListPopover/component.js]: Done. I have added one more nesting as we decided.

File: src/pages/Movie/TopInfo/AddToListPopover/container.js (hunk 2 above)
Comment (pink-bunny) [reply src/pages/Movie/TopInfo/AddToListPopover/container.js]: Done.

File: src/pages/Movie/TopInfo/AddToListPopover/AddToListButton/container.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import AddToListButtonComponent from './component';
+
+class AddToListButton extends React.Component {
+  handleCLick = () => {
+    const { id, addToList } = this.props;
+    addToList(id);
```
Comment (plzen) [inline src/pages/Movie/TopInfo/AddToListPopover/AddToListButton/container.js]: That's exactly what I meant in my comment 👍

**Pull request merged**